OWML.ModHelper.Menus/OwmlConfigMenu.cs

```diff
@@ -0,0 +1,53 @@
+﻿using OWML.Common;
+
+namespace OWML.ModHelper.Menus
+{
+    public class OwmlConfigMenu : ModConfigMenu
+    {
+        private const string GamePathTitle = "Game path";
+        private const string VerboseModeTitle = "Verbose mode";
+        private const string BlockInputTitle = "Block combination input";
```

Comment:
Is this right, @TAImatem ?

Reply:
maybe rather "Block input on combination" or smth like that?

Reply:
Does combination mean collision - "Block input on collision"?

Reply:
this option is for InputHandler's function for blocking game's InputCommands when combination is detected

Reply:
@TAImatem @amazingalek In Mod Manager I called it "Mod button combinations block game input". I think that makes it pretty clear, but maybe too long of a text for the game's menus?

Reply:
Surprisingly it fits :)